BranchMetrics/ios-branch-deep-linking-attribution
# SDK 2005 iOS placeholder stop calling v1 profile and v1 logout

## Reference
SDK-2005 -- Stop calling /v1/profile and /v1/logout.

## Summary
Store and clear the identity locally without making requests to the endpoints.

## Type Of Change
- [ ] New feature (non-breaking change which adds functionality) --( Its not a breaking feature right now )

## Testing Instructions
-> All unit tests should pass
-> Following scenarios should work-
 *  User should be able to set identity and then retrieve it later.
 *  User should get back first referring params via callback.
 *  User should be able to logout. On logout only identity is cleared. First referring params and latest referring params are not cleared.
 * Ensure No network calls are made for endpoints /v1/profile and v1/logout. 

cc @BranchMetrics/saas-sdk-devs for visibility.
